### ISSUE TITLE
This support L7 tracing ? 

### ISSUE THREAD

**Issue by ghtian123** [kmesh-net/kmesh#663]
**Please provide an in-depth description of the question you have**:

**What do you think about this question?**:

**Environment**:
- Kmesh version:
- Others:

**Comment by hzxuzhonghu** [comment]
Which mode do you use? Ads or workload

**Comment by ghtian123** [comment]
> Which mode do you use? Ads or workload

@hzxuzhonghu  workload mode

**Comment by ghtian123** [comment]
Can I replace ztunnel with this?

@hzxuzhonghu

**Comment by YaoZengzeng** [comment]
@ghtian123 According to our analysis, waypoint for L7 tracing is not available, even in Ambient.

**Comment by hzxuzhonghu** [comment]
>Can I replace ztunnel with this?

Yes, the workload mode, actuially is a Layered mode, we replace ztunnel with ebpf, which is more efficient
